### PR TITLE
Add Chinese (Traditional) Translations.

### PR DESCRIPTION
**Pull request by pan93412**

### CODE REVIEW

File: shared/langs/zh-tw.php [hunk 1]
```diff
@@ -1,158 +1,161 @@
...
-$s['latestFetchLatest'] = 'Fetch the latest build';
-$s['latestTestingOnly'] = 'Testing purposes only';
-$s['latestTestingOnlyWarn'] = '<b>This page is provided for testing purposes only.</b> Builds retrieved by this page that were not processed by the backend server will be provided using fallback packs, which may provide incomplete results. If you want to download an already known build, please use the known builds page instead.';
-$s['latestDoYouWantKnown'] = 'Would you like to continue by browsing the list of known builds?';
-$s['chooseOptions'] = 'Choose options';
-$s['buildOfPretendedClient'] = 'Build number of mock Windows Update client';
-$s['editionOfPretendedClient'] = 'Edition of mock system';
+$s['latestFetchLatest'] = '取得最新組建';
+$s['latestTestingOnly'] = '僅供測試用途';
+$s['latestTestingOnlyWarn'] = '<b>這個頁面僅供測試用途。</b> 從這個頁面取得之後端伺服器未處理的組建版本 server will be provided using fallback packs, which may provide incomplete results. If you want to download an already known build, please use the known builds page instead.';
```
Comment (pan93412) [inline shared/langs/zh-tw.php]: i18n: complete the unfinished translations.

File: shared/langs/zh-tw.php [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+//known.php
+$s['browseKnown'] = '瀏覽已知組建';
+$s['chooseBuild'] = '選擇組建';
+$s['weFoundBuilds'] = '我們已依您的搜尋內容找到 <b>%d</b> 個組建。'; //We have found <b>692</b> builds for your query.
+
+//latest.php
+$s['latestFetchLatest'] = '取得最新組建';
+$s['latestTestingOnly'] = '僅供測試用途';
+$s['latestTestingOnlyWarn'] = '<b>這個頁面僅供測試用途。</b> 從這個頁面取得之後端伺服器未處理的組建版本 server will be provided using fallback packs, which may provide incomplete results. If you want to download an already known build, please use the known builds page instead.';
+$s['latestDoYouWantKnown'] = '請問你是否要瀏覽已知組件列表？';
```
Comment (pan93412) [inline shared/langs/zh-tw.php]: i18n: 組件 to 組建

File: shared/langs/zh-tw.php [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+$s['chooseOptions'] = '選擇選項';
+$s['buildOfPretendedClient'] = 'mock Windows Update 客戶端的組建號碼';
+$s['editionOfPretendedClient'] = 'mock 系統版本';
+$s['skipAheadLabel'] = 'Skip ahead flight';
```
Comment (pan93412) [inline shared/langs/zh-tw.php]: l10n: tag `FUZZY`

File: shared/langs/zh-tw.php [hunk 4]
```diff
@@ -1,158 +1,161 @@
...
-$s['allLangsWarn'] = 'The <i>All languages</i> option does not support edition selection.';
-$s['clickNextToOpenFindFiles'] = 'Click the <i>Next</i> button to open the page, which allows finding files.';
-$s['noLangsAvailable'] = 'There are no languages available for this build.';
-$s['browseFiles'] = 'Browse files';
-$s['browseFilesDesc'] = 'Quickly browse files in selected build';
-$s['searchFiles'] = 'Search files';
-$s['toSearchForCUUseQuery'] = 'To search for Cumulative Updates use the <i>%s</i> search query.'; //To search for Cumulative Updates use the <i>Windows10 KB</i> search query.
+$s['selectLangFor'] = '選擇 %s 的語言'; //Select language for Windows 10 Insider Preview 18890.1000 (rs_prerelease) amd64
+$s['chooseLang'] = '選擇語言';
+$s['chooseLangDesc'] = '請選擇你想要的語言';
```
Comment (pan93412) [inline shared/langs/zh-tw.php]: l10n: All of 你 should be 您

File: shared/langs/zh-tw.php [hunk 5]
```diff
@@ -1,158 +1,161 @@
...
+$s['aria2Opt2'] = '使用 aria2 下載並轉換';
+$s['aria2Opt2Desc'] = '使用 aria2 簡單地下載選取的 UUP 集合並轉換成 ISO。';
+$s['aria2Opt3'] = '使用 aria2 下載、轉換並建立額外版本';
+$s['aria2Opt3Desc'] = '使用 aria2 簡單地下載選取的 UUP 集合，再轉換及建立額外版本，最後建立 ISO 映像。';
+$s['jsRequiredToConf'] = '設定及使用此選項需要 JavaScript。';
+$s['selAdditionalEditions'] = '選擇額外版本';
+$s['noAdditionalEditions'] = '此選取項目沒有可供使用的額外版本。';
+$s['learnMore'] = '了解更多';
+$s['learnMoreAdditionalEditions1'] = '這個選項會啟用自動建立選擇的額外版本。';
+$s['learnMoreAdditionalEditions2'] = '這個額外版本列表基於選取的基礎版本。你能在底下查看要建立所需額外版本所需要的版本：';
```
Comment (pan93412) [inline shared/langs/zh-tw.php]: 建立「想要的」版本

File: shared/langs/zh-tw.php [hunk 6]
```diff
@@ -1,158 +1,161 @@
...
-$s['totalSizeOfFiles'] = 'Total size of files: %s'; //Total size of files: 2.86 GiB
-$s['fileRenamingScript'] = 'File renaming script';
-$s['fileRenamingScriptDesc1'] = 'The script found below can be used to quickly rename downloaded files.';
-$s['fileRenamingScriptDesc2'] = 'Simply copy the contents of the form below to a new file with <code>cmd</code> extension, put it in folder with downloaded files and run.';
-$s['sha1File'] = 'SHA-1 checksums file';
-$s['sha1FileDesc'] = 'You can use this file to quickly verify that files were downloaded correctly.';
+$s['listOfFilesFor'] = '列出 %s 的檔案'; //List of files for Windows 10 Insider Preview 18890.1000 (rs_prerelease) amd64
+$s['totalSizeOfFiles'] = '檔案總計大小: %s'; //Total size of files: 2.86 GiB
+$s['fileRenamingScript'] = '檔案重新命名指令碼';
+$s['fileRenamingScriptDesc1'] = '下方所找到的指令碼可用來快速地重新命名下載檔案。';
```
Comment (pan93412) [inline shared/langs/zh-tw.php]: fuzzy

File: shared/langs/zh-tw.php [hunk 7]
```diff
@@ -1,206 +1,210 @@
...
-$s['error_UNKNOWN_ARCH'] = 'Unknown processor architecture.';
-$s['error_UNKNOWN_RING'] = 'Unknown ring.';
-$s['error_UNKNOWN_FLIGHT'] = 'Unknown flight.';
+$s['error'] = '錯誤';
+$s['requestNotSuccessful'] = '請求失敗';
+$s['anErrorHasOccurred'] = '嘗試處理您的請求時發生錯誤。';
+$s['generatedPackNotAvailable'] = '沒有可用的產生套件';
+// TRANTAG: [Fuzzy]
+$s['generatedPackNotAvailableDesc'] = '你嘗試下載的更新尚無會提供關於可用語言、版本及檔案完整資訊的產生套件。將會使用備用套件，可能無法提供正確的資訊。如因這個原因而下載失敗，請等待自動產生套件可用。';
+$s['arm64Warning'] = '您選擇了<b>只支援基於 ARM64 之裝置的</b> ARM64 組建，可能無法在一般基於 Intel 或 AND 的 PC 上運作。若要用於<b> 64 位元</b>的 PC，請使用 <b>amd64</b> 組建。若要用於<b> 32 位元</b>的 PC，請使用 <b>x86</b> 組建。如果你百分之百確定目標裝置基於 ARM64，則您可安全地忽略此訊息。';
```
Comment (pan93412) [inline shared/langs/zh-tw.php]: `AND` PC -> AMD.

File: shared/langs/zh-tw.php [hunk 8]
```diff
@@ -1,206 +1,210 @@
...
-$s['error_KEY_NOT_EXISTS'] = 'Specified key does not exist in update information.';
-$s['error_UNSPECIFIED_UPDATE'] = 'Update ID was not specified.';
-$s['error_INCORRECT_ID'] = 'Specified Update ID is incorrect. Please make sure that the specified Update ID is correct.';
-$s['error_RATE_LIMITED'] = 'You are being rate limited. Please try again in a few seconds.';
-$s['error_UNSPECIFIED_VE'] = 'You have not selected any additional editions. If do not wish to create additional editions, please use the <i>Download using aria2 and convert</i> option.';
-$s['errorNoMessage'] = 'Error message unavailable.';
+$s['error_ILLEGAL_BUILD'] = '指定的組建號碼小於 %d 或大於 %d。'; //Specified build number is less than 9841 or larger than 2147483646.
+$s['error_ILLEGAL_MINOR'] = '指定的組建次要版本不正確。';
+$s['error_NO_UPDATE_FOUND'] = '伺服器沒有回傳更新。';
+$s['error_XML_PARSE_ERROR'] = '解析 XML 回應失敗。可能是因為 Microsoft 伺服器出現問題。請稍候再試。';
```
Comment (pan93412) [inline shared/langs/zh-tw.php]: 發生問題

File: shared/langs/zh-tw.php (hunk 5 above)
Comment (pan93412) [reply shared/langs/zh-tw.php]: 所需 is allowed.

**Comment by pan93412** [comment]
@whatever127 Ok, this pull request is available to merge! Please ask me freely if you found any mistakes.

**Comment by whatever127** [comment]
I did a quick review using Google Translate (since I don't even know a single Chinese character) and everything it seemed OK for me. Please translate other language names in the file and I'll merge it your addition.

**Comment by pan93412** [comment]
@whatever127 Oh, thanks for your review. I'll translate those languages as soon as possible :)

**Comment by pan93412** [comment]
@whatever127 I don't think that translating those language names is great, for some users who never uses Chinese as their native language, they must be confused when they want to switch to their native language after clicking a wrong choice. So, I just preserve the original language name. You can give me the other idea about this decision.

**Comment by whatever127** [comment]
> I don't think that translating those language names is great, for some users who never uses Chinese as their native language, they must be confused when they want to switch to their native language after clicking a wrong choice. So, I just preserve the original language name. You can give me the other idea about this decision.

These language names are used on build selection pages. This is how it looks with current translation:
image

Language names on the language selection of page use native names of languages. This is how it looks like with your translation applied:
image

If you need explanation for other things, feel free to ask.

**Comment by pan93412** [comment]
Ok, thanks for explaining. I'll translate those as soon as possible.

**Comment by pan93412** [comment]
@whatever127 Done, could you have a look? thanks! :)

**Comment by whatever127** [comment]
It seems that everything is OK. Thanks.

I'm merging this now. If you will find some problems then feel free to post a new pull request.

**Pull request merged**